### PR TITLE
Added wrist Mk 2.1 project

### PR DESCRIPTION
**Pull request by ale-git**
I've created the Simscape digital twin of the Mk 2.1 spherical wrist.
The wrist_decoupler subsystem used by the Simulink controller is used to automatically generate the code that is included in the EMS and AMC firmware projects, and it is now reconfigurable at runtime, i.e. it hasn't to be updated or regenerated when there is a change in the geometric structure of the wrist. The configurable parameters at runtime are:
- arm angle offests
- platform angle offsets
- arm bending
- right/left wrist

### CODE REVIEW

**Comment by pattacini** [comment]
Thanks heaps @ale-git 👍🏻 
I'll have a look asap.

File: wrist_mk2_1/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Wrist Mk2
```
Comment (pattacini) [inline wrist_mk2_1/README.md:1]:
```suggestion
Wrist Mk2.1
```

File: wrist_mk2_1/README.md
Comment (pattacini) [inline wrist_mk2_1/README.md:1]: I've noticed that images and text are a verbatim copy of wrist mk2.

I'd suggest that we update the content of the README in order to point out the part on the configuration introduced by this PR.

**Review by pattacini — CHANGES_REQUESTED** [review]
Thanks @ale-git for this upgrade.

I've posted a few inline comments to address for the README.

Also, a couple of questions:
1. Since this model subsumes mk2 (if I got it right), do you think that in the long run we could remove the old model, keeping it available just in the short period?
2. Have you already planned to see if the current [documentation](https://icub-tech-iit.github.io/documentation/wrists/) deserves some amendments?

**Comment by pattacini** [comment]
>Since this model subsumes mk2 (if I got it right), do you think that in the long run we could remove the old model, keeping it available just in the short period?

Just aligned F2F w/ @ale-git. The mechanics are different and could be handled easily within a unique model relying on variant subsystems.

**Review by ale-git — COMMENTED** [review]
Hi @pattacini, I've updated the README.md with the new pictures and detail of the geometry configuration.

**Review by pattacini — APPROVED** [review]
Thanks!
Merging.

**Pull request merged**